draft-ietf-quic-tls.md

```diff
@@ -1567,12 +1573,13 @@ QUIC requires that the cryptographic handshake provide authenticated protocol
 negotiation.  TLS uses Application Layer Protocol Negotiation (ALPN)
 {{!ALPN=RFC7301}} to select an application protocol.  Unless another mechanism
 is used for agreeing on an application protocol, endpoints MUST use ALPN for
-this purpose.  When using ALPN, endpoints MUST immediately close a connection
-(see Section 10.3 in {{QUIC-TRANSPORT}}) if an application protocol is not
-negotiated with a no_application_protocol TLS alert (QUIC error code 0x178, see
-{{tls-errors}}).  While {{!ALPN}} only specifies that servers use this alert,
-QUIC clients MUST also use it to terminate a connection when ALPN negotiation
-fails.
+this purpose.
+
+When using ALPN, endpoints MUST immediately close a connection (see Section
+10.3 of {{QUIC-TRANSPORT}}) if an application protocol is not negotiated with a
+no_application_protocol TLS alert (QUIC error code 0x178, see {{tls-errors}}).
```

Comment:
```suggestion
10.3 of {{QUIC-TRANSPORT}}) with a no_application_protocol TLS alert (QUIC
error code 0x178, see {{tls-errors}}) if an application protocol is not negotiated.
```
"close a connection with... if..." makes it clearer which phrase the "with" applies to.